tutorials/advanced/elasticSnorkel.ipynb

```diff
@@ -0,0 +1,583 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Intializing Elasitcsearch\n",
+    "\n",
+    "We begin by intializing Elasticsearch with the names of our index and type. You should have Elasticsearch installed and already run the \"bin/elasticsearch\" command. For more information you can visit: https://www.elastic.co/guide/en/elasticsearch/guide/current/running-elasticsearch.html.\n",
+    "\n",
+    "Elasticsearch uses slightly different terminology from tradational databases. \n",
+    "\n",
+    "This is the following mapping:\n",
+    "Database = Index\n",
+    "Table = Type\n",
+    "Row = Document\n",
+    "\n",
+    "Here we intialize an object where corpus will be the name of the index with a single type articles, and a field named sentence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from snorkel import SnorkelSession\n",
+    "from snorkel.models import Document, Sentence\n",
+    "from elastics import elasticSession ,printResults,deleteIndex\n",
+    "\n",
+    "# import os\n",
+    "\n",
+    "# # TO USE A DATABASE OTHER THAN SQLITE, USE THIS LINE\n",
+    "# # Note that this is necessary for parallel execution amongst other things...\n",
+    "# os.environ['SNORKELDB'] = 'postgres:///snorkel-intro'\n",
+    "\n",
+    "session = SnorkelSession()\n",
+    "eSearch=elasticSession(\"corpus\",\"articles\",\"sentence\")"
```

Comment:
Do they need to provide these args (and what are they- "corpus", "articles"?)  Can we skip this and just assume default context hierarchy?